angular/projects/public-nrpti/src/app/records/records-list/search-filters/search-filters.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="search-filter-container">
+  <section>
+    <div class="filter-header">
+      <span><h3>Filter</h3></span>
+      <button class="btn btn-primary" (click)="closeButton.emit()">Minimize Advanced Filters</button>
+    </div>
+    <div>
+      <p>
+        The results list below will update instantly as you select criteria to filter, the URL will also update with
+        your unique criteria. If you want to return to your exact criteria, you can bookmark the page once you have
+        filtered (note that results may not be identical as content is updated).
+      </p>
+    </div>
+  </section>
+  <section>
+    <form *ngIf="!loading" [formGroup]="formGroup" novalidate>
+      <div class="flex-container date-container no-wrap">
+        <div class="label-pair">
+          <label for="recordType">Issued Date</label>
+          <lib-date-picker [control]="formGroup.get('dateIssuedStart')" [reset]="resetControls"></lib-date-picker>
+        </div>
+        <span class="date-separator">To</span>
+        <div class="label-single">
+          <lib-date-picker [control]="formGroup.get('dateIssuedEnd')" [reset]="resetControls"></lib-date-picker>
+        </div>
+      </div>
+      <div class="flex-container issuedTo-container no-wrap">
+        <div class="label-pair">
+          <label for="issuedTo">Issued To</label>
+          <span></span>
```

Comment:
What's this?

Reply:
No idea, probably a half finished thing in my haste to get this out the door.

Removed!